src/components/MovieBoard/MovieBoard.js

```diff
@@ -22,6 +21,5 @@ import PropTypes from "prop-types"
 export default MovieBoard
 
 MovieBoard.propTypes = {
-  movies: PropTypes.array,
-  selectMovie: PropTypes.func
+  movies: PropTypes.array
```

Comment:
Is this prop required? If so, consider adding `isRequired`

Reply:
Just added!